contracts/nois-proxy/src/state.rs

```diff
@@ -33,6 +33,8 @@ pub struct Config {
     /// The prices of a randomness. List is to be interpreted as oneof,
     /// i.e. payment must be paid in one of those denominations.
     pub prices: Vec<Coin>,
+    /// Manager to set the config
+    pub manager: Addr,
```

Comment:
okay, this will make upgrading existing proxies tricky. You need a state migration in the migradte entry point. Do you want to have this migration? Could be a nice excercise.